fastlane_core/lib/fastlane_core/device_manager.rb

```diff
@@ -205,6 +226,7 @@ def copy_logarchive(device, dest)
         command = "xcrun simctl spawn #{device.udid} log collect 2>/dev/null"
         FastlaneCore::CommandExecutor.execute(command: command, print_all: false, print_command: true)
 
+        FileUtils.mkdir_p(dest)
```

Comment:
this causes troubles, on all machines i am able to test this on.

results in a `system_logs-iPhone 6s_iOS_10.2.logarchive` which is a REAL folder - and if open it with console app its empty, if i do `Show package contents`  inside the `system_logs-iPhone 6s_iOS_10.2.logarchive` there is a `system_logs.logarchive`